project/src/components/main-screen/main-screen.tsx

```diff
@@ -74,7 +78,12 @@ function MainScreen({placesCount}: MainScreenProps): JSX.Element {
           </section>
         </div>
         <div className="cities">
-          {placesCount? <PlacesList placesCount={placesCount} /> : <PlacesListEmpty />}
+          <div className={clsx('cities__places-container container', !offers.length && 'cities__places-container--empty')}>
+            {offers.length ? <PlacesList offers={offers} onActiveHandler={handleActiveOffer} /> : <PlacesListEmpty />}
+            <div className="cities__right-section">
+              <Map city={offers[0].city} points={points} selectedPoint={selectedPoint} />
```

Comment:
[nip] - возможна потенциальная runtime ошибка:
* `offers[0]` может вернуть undefined в связи c чем приложение упадет с ошибкой `Uncaught TypeError: Cannot read properties of undefined (reading 'city')` 
* решение - воспользоваться оператором `optional chaining` 
```suggestion
              <Map city={offers[0]?.city} points={points} selectedPoint={selectedPoint} />
```